bin/desi_tsnr_afterburner

```diff
@@ -57,6 +61,12 @@ def parse(options=None):
                         help = 'Ouput tile completeness table')
     parser.add_argument('--aux', type = str, default = None, required=False, nargs="*",
                     help = 'Path to auxiliary ttiles ables, like /global/cfs/cdirs/desi/survey/observations/SV1/sv1-tiles.fits')
```

Comment:
for our future selves, please add a note about when this is needed and/or consequence of not providing aux.

Reply:
added message in help.